databroker/broker.py

```diff
@@ -170,6 +172,56 @@ def __iter__(self):
                         break
 
 
+CONFIG_SEARCH_PATH = (os.path.expanduser('~/.config/databroker/'),
```

Comment:
Do we want to special-case conda here? We did that previously, but it retrospect it seems like kind of a hacked thing to do. Is there any reason we need it, given our plans for the config deployment?